go.mod

```diff
@@ -240,3 +248,57 @@ replace (
 	golang.org/x/net => golang.org/x/net v0.0.0-20210510120150-4163338589ed
 	golang.org/x/sys => golang.org/x/sys v0.0.0-20220114195835-da31bd327af9
 )
+
+replace k8s.io/api => k8s.io/api v0.25.2
```

Comment:
The version is same with what you import above.
I guess there is no need to do replace here.

Reply:
If not replace,it will cause this error:
![image](https://user-images.githubusercontent.com/56665618/193205969-23427a79-0eb0-4421-bab9-3ea163799163.png)
This is because in Kubernetes go.mod, you'll find the following code:
![image](https://user-images.githubusercontent.com/56665618/193206471-ed9bbd83-2f72-4409-9e43-b317215a8071.png)
In the Kubernetes repository, the packages listed under the require commands are marked as v0.0.0